### PR TITLE
fix: added second command in order to build the applicaiton

### PR DESCRIPTION
**Pull request by bonjo7**
#### Please check if the PR fulfills these requirements

- [x] The commit message is sensible and easily understood
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

#### What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)

**Fix:** presently there is an issue with running `npm run build` as the build constantly fails

In the mean time I have added `npm run clean` to the `build` command as this will allow developers to run the application locally without an error issues

Another small change is to the readme file as currently the path is wrong to copy the env file

#### Thank you for your contribution

We hope you stay around and connect with our growing community!

### CODE REVIEW

**Comment by edeleastar** [comment]
Thanks @bonjo7 - very helpful. 
The root of the issue I think is the presence of tsconfig.buildinfo files in the repo, which I have now removed. Nevertheless, an additional clean is a good idea. More work to be done, I think, on the components and apps configurations yet.

File: README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ cd sites/tutors-course-reader
 and copy the file `.env.example` to `.env`
 
 ```bash
-cp .\.env.example .env
```
Comment (lgriffin) [inline README.md:65]: @bonjo7 ./ is a general notation for "look in this directory" -- yours works for sure but if someone tabs the OS tends to put ./ before it.

Comment (bonjo7) [reply README.md:65]: Ah ok, gotcha @lgriffin , so should it not be `./.env...` and not `.\.env...`

Forward slash `/` rather than back slash `\`

Comment (lgriffin) [reply README.md:65]: Haha actually I didn't even cop it as I done the PR to add it from my Windows machine but you are right they are flipped on Linux / Unix based systems.

**Pull request closed without merging**